packages/arb-token-bridge-ui/src/components/syncers/BalanceUpdater.tsx

```diff
@@ -1,17 +1,26 @@
 import { useEffect } from 'react'
 import { useLatest } from 'react-use'
+import { useAccount } from 'wagmi'
 
 import { useAppState } from '../../state'
+import { useUpdateUSDCBalances } from '../../hooks/CCTP/useUpdateUSDCBalances'
 
 // Updates all balances periodically
 const BalanceUpdater = (): JSX.Element => {
   const {
     app: { arbTokenBridge, selectedToken }
   } = useAppState()
+  const { address: walletAddress } = useAccount()
   const latestTokenBridge = useLatest(arbTokenBridge)
 
+  const { updateUSDCBalances } = useUpdateUSDCBalances({
+    walletAddress
+  })
+
   useEffect(() => {
     const interval = setInterval(() => {
+      updateUSDCBalances(selectedToken?.address)
```

Comment:
We need to run this even if a token is not selected to always update USDC balances and have them reflected in the token search.